core/objects.go

```diff
@@ -143,14 +143,10 @@ type AcmeIdentifier struct {
 // CertificateRequest is just a CSR together with
 // URIs pointing to authorizations that should collectively
 // authorize the certificate being requsted.
-//
-// This type is never marshaled, since we only ever receive
```

Comment:
The CSR and Authorizations are both received from the client, so there's no extra information on this struct. Referencing rawCertificateRequest is a good call though, I'll add that.